packages/smooth_app/lib/generic_lib/buttons/smooth_large_button_with_icon.dart

```diff
@@ -58,6 +58,8 @@ class SmoothLargeButtonWithIcon extends StatelessWidget {
             child: AutoSizeText(
```

Comment:
Perhaps adding an additional `overflow` like `ellipsis` would also look better than the text being cut, anyway?

Reply:
Ok